src/selectize.js

```diff
@@ -406,14 +406,15 @@ $.extend(Selectize.prototype, {
 		} else {
 			// If a regex or string is included, this will split the pasted
 			// input and create Items for each separate value
-			if (self.settings.splitOn) {
-				setTimeout(function() {
+			setTimeout(function() {
+				if (self.settings.splitOn) {
```

Comment:
I believe it's just to make `self.onKeyUp(e)` (like 461) always be triggered inside the delayed function. The issue is not with order of `setTimeout` and `if`, but in the fact that `self.onKeyUp(e)` also needs to be triggered after a delay.